### PR TITLE
Update 'tommath' to v1.0.1

### PR DESCRIPTION
**Pull request by t0p4**

### CODE REVIEW

File: bn.tex [hunk 1]
```diff
@@ -0,0 +1,1913 @@
+\documentclass[synpaper]{book}
```
Comment (ruslo) [inline bn.tex:1]: Do we really have to touch this file?

File: booker.pl [hunk 2]
```diff
@@ -0,0 +1,267 @@
+#!/bin/perl
```
Comment (ruslo) [inline booker.pl:1]: Same here, probably this part can be reverted (?)

**Review by ruslo — CHANGES_REQUESTED** [review]

File: bn.tex (hunk 1 above)
Comment (t0p4) [reply bn.tex:1]: no, we don't

File: booker.pl (hunk 2 above)
Comment (t0p4) [reply booker.pl:1]: yes, you're right

File: bn.tex (hunk 1 above)
Comment (ruslo) [reply bn.tex:1]: > no, we don't

Okay, great. Can you please remove it from commit then?

**Review by ruslo — CHANGES_REQUESTED** [review]

File: bn.tex (hunk 1 above)
Comment (t0p4) [reply bn.tex:1]: How to do it right?

Comment (t0p4) [reply bn.tex:1]: Do I need to start over from first step in guide ?

Comment (ruslo) [reply bn.tex:1]: > Do I need to start over from first step in guide ?

You mean Hunter documentation? No :) It's a pure Git task.

> How to do it right?

I guess there are a lot of ways but if I will do similar thing I will:
* create another branch for pull request
* switch to new branch
* cherry-pick commits one by one
* stop on the commit that introduce `bn.tex`, do `git rm bn.tex` and `git commit --amend`, effectively this step will remove `bn.tex`
* cherry-pick the rest

**Pull request closed without merging**